Resources/public/js/cookie_consent.js

```diff
@@ -37,7 +37,8 @@ document.addEventListener("DOMContentLoaded", function() {
                 xhr.open('POST', cookieConsentForm.action);
                 xhr.setRequestHeader('Content-Type', 'application/x-www-form-urlencoded');
                 xhr.send(serializeForm(cookieConsentForm, event.target));
-		// Clear all styles from body to prevent the white margin at the end of the page
+                location.reload();
+                // Clear all styles from body to prevent the white margin at the end of the page
```

Comment:
I believe this will prompt the user to re-submit any forms (if they have sent any POST data recently), which is not an optimal UX. I think `window.location.href = window.location.href;` should work. Also, the document styles below will become redundant because they will not happen.

Reply:
Good suggestion, I can use that in the event listener. For now it creates a custom event, instead of a forced refresh after a button click, so everyone can use another functionality if required. (see new commit)